infrastructure/helm/quantumserverless/values.yaml

```diff
@@ -9,6 +9,28 @@
 # Ingress Nginx controller is disabled by default to not affect cloud providers' controller configuration
 nginxIngressControllerEnable: true
 
+# ===================
+# Gateway configs
+# ===================
+
+gatewayEnable: true
+gateway:
+  image:
+    pullPolicy: Always
+  application:
+    siteHost: "http://127.0.0.1:8000"
+    rayHost: "http://serverless-cluster-kuberay-head-svc:8265"
+    keycloak:
+      clientId: "gateway-client"
+      url: "http://serverless-cluster-keycloak/"
```

Comment:
Is `serverless-cluster` your helm install release name?  Would you replace it with HELM-RELEASE